### PR TITLE
Add metrics related to wallet

### PR DESCRIPTION
**Pull request by Partysun**
## Changes

- Historical balance
- Latest Transactions
- Top transfers
- Contract Transactions Count
- Contract Interacting Address Count

## Checklist

- [x] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/metrics/contract-interacting-address-count/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+author: Yury
+date: 2022-04-21
+description: Get interacting address count for a chain address / contract
+---
+
+## Definition
+
+Get interacting address count for a chain address / contract
+
+[Example of usage **interacting address count** on Sanbase](https://app.santiment.net/s/rk-8It-D)
```
Comment (IvanIvanoff) [inline src/docs/metrics/contract-interacting-address-count/index.md:12]: I don't think it's a good idea to add links to personal chart layouts. We also don't do this in other places.
Furthermore, these metrics are *very* slow to compute, so even if you click - nothing happens for long period of time.

In this case, I think screenshots would work better. And if we have link to charts, they should not be to personal stuff but better to be a short URL without a chart layout.

Comment (Partysun) [reply src/docs/metrics/contract-interacting-address-count/index.md:12]: @IvanIvanoff You are right. Thank you. Will update it.

File: src/docs/metrics/contract-interacting-address-count/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Query variables:
+```json
+{
+  "address": "0x857086e5e3dc7bbc98edb1639b4ffd96a667d75a",
+  "from": "2021-10-21T18:59:59.999Z",
+  "interval": "4h",
+  "metric": "contract_interacting_addresses_count",
+  "to": "2022-04-21T18:59:59.999Z"
+}
+```
```
Comment (IvanIvanoff) [inline src/docs/metrics/contract-interacting-address-count/index.md:46]: Please follow the structure in the other metric examples. Providing query and arguments separately makes the query much less readable.

File: src/docs/metrics/contract-transactions-count/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Access
+
+[Free Access](/metrics/details/access#free-access)
+
+---
+
+## API
+
+Available under the `Transactions Count` name:
```
Comment (IvanIvanoff) [inline src/docs/metrics/contract-transactions-count/index.md:24]: The name in the API is `transactions_count` (no spaces, underscore, all lowercase)

File: src/docs/metrics/historical-balance/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: Historical Balance
+author: Yury
+date: 2022-04-21
+description: Metric returns historical balance for a chain address / contract
+---
+
+## Definition
+
+Metric returns historical balance for a chain address / contract
```
Comment (IvanIvanoff) [inline src/docs/metrics/historical-balance/index.md:10]: We'll need a separate page in the /details that has a description of what `balance` is and link it here. Ideally, definition should provide more information than the one found in the name of the metric.

**Pull request merged**